### DISCUSSION TITLE
Message logging

### DISCUSSION THREAD

**Discussion by WiseSewaTroll** [CoreWCF/CoreWCF#923] [Q&A] [answered]
Hi,
Is diagnostic (message) logging supported in CoreWCF? I have added configuration for it and am using AddServiceModelConfigurationManagerFile, but this configuration seems to be ignored?

**Comment by g7ed6e** [comment, marked as answer]
Hello @WiseSewaTroll message logging is not implemented yet in CoreWCF. In the mean time, depending of what processing you want to execute you can however rely on one of the solutions mentioned in #809.

**Reply by WiseSewaTroll** [reply]
Thanks for the quick response. Went with the middleware approach in order to log the full soapenv.
